drivers/source/usb/USBCDC_ECM.cpp

```diff
@@ -556,3 +556,4 @@ void USBCDC_ECM::_bulk_out_callback()
 
     read_start(_bulk_out, _bulk_buf, MAX_PACKET_SIZE_BULK);
 }
+#endif // defined(MBED_CONF_RTOS_PRESENT)
```

Comment:
Newline missing

Reply:
can you quickly update the file and we restart CI?

Reply:
Done. Please restart the CI